src/api/utils.ts

```diff
@@ -25,3 +31,25 @@ export const handleApiError = <T>(error: AxiosError<T>, onError?: OnError) => {
     if (onError) return onError(error);
   }
 };
+
+type getCellType = (row: IRow, index: number | undefined) => string | HumanizedSortable;
+
+export const rowSorter = (sortBy: ISortBy, getCell: getCellType) => (a: IRow, b: IRow): number => {
```

Comment:
Would it make more sense to move this into `components/ui/table/utils.ts`?

Reply:
done